x-pack/test_serverless/functional/test_suites/observability/observability_logs_explorer/alerts.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['observabilityLogsExplorer']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Alerts dropdown menu', () => {
+    before(async () => {
```

Comment:
**question:** does this pass correctly? Does not require you to perform any login step?

Reply:
Yeh, it didn't pass :/ 
It's strange, the on-prem tests don't seem to require a login step.

Reply:
Yes, they have different authentication services IIRC, your last change should fix it 👌